tests/php/VersionedGridFieldItemRequestTest/TestExtension.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\Versioned\Tests\VersionedGridFieldItemRequestTest;
+
+use SilverStripe\Core\Extension;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\FormAction;
+
+/**
+ * Test extension that adds actions via updateFormActions
+ */
+class TestExtension extends Extension implements TestOnly
+{
+    /**
+     * @var bool Flag to control whether to add actions
+     */
+    public static $add_test_action = false;
+
+    public function updateFormActions(FieldList $actions)
+    {
+        if (self::$add_test_action) {
```

Comment:
```suggestion
        if (static::$add_test_action) {
```
We don't use `self` unless we absolutely have to - and in this case `static` is more appropriate since this isn't referencing `private` API or a `final` class.

I'm surprised the linter didn't pick up on this.